Source/WebKitLegacy/mac/WebView/WebViewPrivate.h

```diff
@@ -560,9 +560,6 @@ Could be worth adding to the API.
 - (BOOL)_dashboardBehavior:(WebDashboardBehavior)behavior;
 #endif
 
-+ (void)_setShouldUseFontSmoothing:(BOOL)f;
```

Comment:
How do WK1 tests work?

Reply:
That was harder to find :)

Looks like DumpRenderTree uses NSUserDefaults to set "AppleFontSmoothing":0 for the process.

It looks like WebKitTestRunner tries to do the same thing via the injected bundle, but it's documented as not really working - https://searchfox.org/wubkat/rev/7111cbd9fd80f1b153ea0d0a1805420bae87211c/Tools/TestWebKitAPI/mac/InjectedBundleControllerMac.mm#41-48

Reply:
does that mean we can get rid of the SPI entry point entirely?

Reply:
I think we'd need a way to get the NSUserDefault settings to propagate to all of our WebProcesses in WK2 (which the InjectedBundleControllerMac implementation is sort of trying to do, but not succeeding).